modules/programs/ghostty.nix

```diff
@@ -0,0 +1,171 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.programs.ghostty;
+
+  keyValueSettings = {
+    listsAsDuplicateKeys = true;
+    mkKeyValue = lib.generators.mkKeyValueDefault { } " = ";
+  };
+  keyValue = pkgs.formats.keyValue keyValueSettings;
+in {
+  meta.maintainers = [ lib.maintainers.HeitorAugustoLN ];
+
+  options.programs.ghostty = {
+    enable = lib.mkEnableOption "Ghostty";
```

Comment:
why was this capitalized? it wasn't in a previous commit. i'm not sure how it would affect things, if at all.

Reply:
It doesn't affect anything, it just capitalizes the Ghostty in the description of the option.